_acs-aem-commons/features/package-garbage-collector/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: acs-aem-commons_feature
+title: Package Garbage Collector
+description: Cleans up packages deployed by Cloud Manager
+date: 2022-01-27
+redirect_from: /acs-aem-commons/features/package-garbage-collector.html 
+feature-tags: administration
+initial-release: 5.5.3
+last-updated-release: 5.5.3
+tags: aemcs-incompatible
+---
+
+## Purpose
+When you use Adobe Cloud Manager to deploy new code to your environments on Adobe Managed Service instances, it will generate a unique version number for your package, based on the configuration in the pom.xml file. 
+
+The downside of this approach, is that over time, multiple versions of the packages are building up in the /etc/packages part of the repository. Eventually, this leads to the exhaustion of the storage, and in the worst case scenario, left unchecked; repository corruption. In particular, test environments that usually have less storage available than staging or production environments are more susceptible to this problem. 
+
+Now, Adobe Managed Services has all the appropriate monitoring in place to ensure that worst case scenario doesn't happen, but even if it doesn't get that far, clearing up the installed packages is a chore that frequently takes up time with consultants. 
+
+This is also not a issue with AEM as a Cloud Service, because a new image is generated for each deployment.
+
+## Configuration
+By default, the service will do absolutely nothing. In order to have the job run, OSGI configuration needs to be created:
+- **scheduler:** 
+This is a cron expression that gives great flexibility in when and how frequently the garbage collection is run. By default, an empty value here will cause the job to run at 02:30 every day.
+- **groupName:** 
+This is the group name of the packages that you want to remove.
+- **maxAgeInDays:**
+Packages matching the group name that were installed more than this number of days ago will be removed.
```

Comment:
does it distinguish between versions which are currently installed and versions not being installed?